dataset/dictionary/organizations.json5

```
@@ -137,62 +146,79 @@
   {
     en: "Xinyue Kiosk",
     ja: "新月軒",
+    zhCN: "新月轩",
     pronunciationJa: "しんげつけん",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "Third-Round Knockout",
     ja: "三杯酔",
+    zhCN: "三碗不过港",
     pronunciationJa: "さんばいよい",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "Guhua Clan",
     ja: "古華派",
+    zhCN: "古华派",
     pronunciationJa: "こかは",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "Wanwen Bookhouse",
     ja: "万文集舎",
+    zhCN: "万文集舍",
     pronunciationJa: "まんぶんしゅうしゃ",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "Feiyun Commerce Guild",
     ja: "飛雲商会",
+    zhCN: "飞云商会",
     pronunciationJa: "ひうんしょうかい",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "The Crux Fleet",
     ja: "南十字船隊",
+    zhCN: "南十字船队",
     pronunciationJa: "みなみじゅうじせんたい",
     tags: [ "liyue", "organization" ],
     notes: "単に「南十字」(\"The Crux\") と略して表記される場合もある。",
   },
   {
     en: "Alcor",
     ja: "死兆星号",
+    zhCN: "死兆星号",
     pronunciationJa: "しちょうせいごう",
     tags: [ "liyue", "organization", "location" ],
     notes: "Alcor は北斗七星の近くにある星で、日本ではカタカナでアルコルと記されるが、異称として死兆星とも呼ばれる。",
   },
   {
     en: "Yun-Han Opera Troupe",
     ja: "雲翰社",
+    zhCN: "云翰社",
     pronunciationJa: "うんかんしゃ",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "Hanfeng's Ironmongers",
     ja: "寒鋒鉄器",
+    zhCN: "寒风铁器",
     pronunciationJa: "かんほうてっき",
     tags: [ "liyue", "organization" ],
   },
   {
     en: "Heyu Tea House",
     ja: "和裕茶館",
+    zhCN: "和裕茶馆",
+    tags: [ "liyue", "organization" ],
+  },
+  // Note: Recommend add this: (appeared mainly in stories about Yelan)
+  {
+    en: "Yanshang Teahouse",
+    ja: "岩上茶室",
+    zhCN: "岩上茶室",
```

Thanks, let's add it.

```suggestion
  {
    en: "Yanshang Teahouse",
    ja: "岩上茶室",
    pronunciationJa: "がんじょうちゃしつ",
    zhCN: "岩上茶室",
```